### PR TITLE
Remove non used dirs and logrotate rules

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

**Comment by anadahz** [comment]
The update should also be signed.

**Comment by anadahz** [comment]
In 209c917 made the required changes in order for ooniprobe WUI to listen to port 80.

File: updater/versions/update-1.py [hunk 1]
```diff
@@ -197,6 +210,10 @@ def write_lepidopter_update_logrotate():
     with open(LEPIDOPTER_UPDATE_LOGROTATE_PATH, "w") as out_file:
         out_file.write(LEPIDOPTER_UPDATE_LOGROTATE)
 
+def write_lepidopter_update_logrotate():
```
Comment (hellais) [inline updater/versions/update-1.py]: This function is called like the function above.

Comment (hellais) [reply updater/versions/update-1.py]: You then actually need to call the function in the perform_update part of the script.

**Review by hellais — CHANGES_REQUESTED** [review]
Fix function call

**Review by hellais — APPROVED** [review]

**Comment by hellais** [comment]
I pushed a fix for this, merging.

**Pull request merged**